cpp/include/cudf/copying.hpp

```diff
@@ -79,5 +79,78 @@ std::unique_ptr<column> allocate_like(column_view input, size_type size,
  */
 std::unique_ptr<table> empty_like(table_view input_table);
 
+/**
+ * @brief Slices a column_view (including null values) into a set of column_views
+ * according to a set of indices.
+ *
+ * The `slice` function creates `column_view`s from `input` with multiple intervals
+ * of rows using the `indices` values. Regarding the interval of indices, a pair of 
+ * values are taken from the indices vector in a consecutive manner.
+ * The pair of indices are left-closed and right-open.
+ *
+ * The pairs of indices in the vector are required to comply with the following
+ * conditions:
+ * a, b belongs to Range[0, input column size]
+ * a <= b, where the position of a is less or equal to the position of b.
+ *
+ * Exceptional cases for the indices array are:
+ * When the values in the pair are equal, the function returns an empty `column_view`.
+ * When the values in the pair are 'strictly decreasing', the outcome is
```

Comment:
Is it undefined? Won't it throw an exception if `indices[i] > indices[i+1]`?

Reply:
It should have been updated, my bad.